ui/lib/pki/addon/templates/roles/role/edit.hbs

```diff
@@ -1,3 +1,14 @@
+<PageHeader as |p|>
+  <p.top>
+    <Page::Breadcrumbs @breadcrumbs={{this.breadcrumbs}} />
+  </p.top>
+  <p.levelLeft>
+    <h1 class="title is-3" data-test-pki-role-page-title>
+      Edit
+      {{@model.name}}
```

Comment:
Probably a question for design - but I think our pattern has been "Edit { resource}" - so this would be "Edit PKI role"